okhttp/Module.md

```diff
@@ -0,0 +1,7 @@
+# Module okhttp
+
+An HTTP+HTTP/2 client for Android and Java applications.
+
+# Package okhttp3
+
+An HTTP+HTTP/2 client for Android and Java applications.
```

Comment:
What's the utility in repeating these sentences? We should pick one.

Reply:
One's for the module, and you can have any number of packages with their own. Basically the package lines are the replacement for package-info documentation, and the module line is per-module documentation (either a blurb or you could make it fully filled like an embedded README)

Example of multiple packages

```markdown
# Module okhttp

An HTTP+HTTP/2 client for Android and Java applications.

# Package okhttp3

An HTTP+HTTP/2 client for Android and Java applications.

# Package okhttp3.hypothetical

A hypothetical other package doc.

```

<kbd>
  <img src="https://user-images.githubusercontent.com/1361086/55322060-16408000-5430-11e9-8ea5-f882dd5e524a.png">
</kbd>

Reply:
Can we defer that until we actually have a module with multiple packages?

Reply:
That works. Then it looks like this

<kbd>
  <img src="https://user-images.githubusercontent.com/1361086/55375345-a29a8380-54c0-11e9-97cc-419635cf9d4b.png">
</kbd>

Reply:
Or alternatively you could have the package only and not the module. I did modules only in 3dc147c but should have asked first. Lmk which you prefer

<kbd>
  <img src="https://user-images.githubusercontent.com/1361086/55375450-0d4bbf00-54c1-11e9-9331-5697bb71cbdb.png">
</kbd>